opent2t.base-schemas/opent2t.platform-schema.json

```diff
@@ -0,0 +1,71 @@
+{
```

Comment:
All of our other JSON schemas reside in folders named for the schema itself, in addition, OpenT2T's OcfSchemaReader dereferencer will only find the resource if it is in the same named folder.  Instead of opent2t.base-schemas, we should name the folder opent2t.platform-schema